src/gggears/gggears_wrapper.py

```diff
@@ -640,12 +640,23 @@ def __init__(
         )
         # correct for herringbone design
         if herringbone:
-            self.gearcore.z_vals = np.array([0, self.height / 2, self.height])
+            # assuming z_vals[0] is 0, as it should be
```

Comment:
This comment makes me nervous; should there be an exception raised if it's not zero instead?

Reply:
I'm not raising exceptions anywhere in the code, I'm more of a subscriber of the "We're all adults here" pythonism.
I might change my mind later but this is not a good place to start adding exceptions.

Though you're right, I'll look into it to make a more robust solution and consider the min/max of zvals. And I'll add a test.